.eslintignore

```diff
@@ -3,3 +3,7 @@ ios/
 src/locales/_build/
 src/locales/**/*.js
 src/graphql/generated.ts
+src/app/playground/
+src/components/playground/
+rc/design-system/
+rc/components/uikit/
```

Comment:
Missing "s" and maybe we could target just the index barrel file if that had issues with import sorting.